d3a_interface/sim_results/market_summary_info.py

```diff
@@ -0,0 +1,69 @@
+from typing import Dict
+from statistics import mean
+from d3a_interface.utils import key_in_dict_and_not_none
+from d3a_interface.sim_results.results_abc import ResultsBaseClass
+from d3a_interface.sim_results import is_trade_external
+
+
+class MarketSummaryInfo(ResultsBaseClass):
+
+    def __init__(self, should_export_plots):
+        self._should_export_plots = should_export_plots
```

Comment:
AFAI, its not being used, any reason to introduce for this endpoint?

Reply:
It is used [here](https://github.com/gridsingularity/d3a-interface/pull/177/files#diff-f98c72ddf2055f5c936524db4d713bec808118a4bc18b4011c97869d6156a09aR34), line 34, in order to not update the endpoint if the plots should be exported